core/layout_api.php

```diff
@@ -1250,3 +1250,11 @@ function layout_scroll_up_button() {
 	echo '<i class="ace-icon fa fa-angle-double-up icon-only bigger-110"></i>' . "\n";
 	echo '</a>' . "\n";
 }
+
+function layout_login_page_logo() {
+	?>
+	<div class="login-logo">
+		<img src="<?php echo helper_mantis_url( config_get( 'logo_image' ) ); ?>">
+	</div>
```

Comment:
I'm not sure why an `<h1>` tag was used before, but is there any particular reason to change from that to a `<div>` ?

Reply:
> but is there any particular reason to change from that to a <div>?

div is a neutral tag that does not carry on any formatting, and is better suited to apply a css customization to change the positioning outside of the container flow, if needed.

Reply:
Makes sense.